proto/sequencerblockapis/astria/sequencerblock/v1/block.proto

```diff
@@ -151,6 +151,24 @@ message RollupData {
   }
 }
 
+message UpgradeChangeHashesWithProof {
+  // The SHA256 digests of all upgrade changes applied during this block, if an upgrade was
+  // activated at this height.
+  repeated bytes upgrade_change_hashes = 1;
+  // The proof that the change hashes are included in the cometbft block data (if it exists),
+  // specifically the third item in the data field.
+  astria.primitive.v1.Proof proof = 2;
+}
+
+message ExtendedCommitInfoWithProof {
+  // The extended commit info for the block, if vote extensions were enabled at this height.
+  bytes extended_commit_info = 1;
+  // The proof that the extended commit info is included in the cometbft block data (if it
+  // exists). If no upgrade change hashes exist in the block, this will be the third item in the
+  // cometbft block data field, otherwise it will be fourth.
```

Comment:
move the "If no upgrade change hashes exist" part to above `Message ExtendedCommitInfoWithProof`?

Reply:
Done in [4a045d3](https://github.com/astriaorg/astria/pull/1917/commits/4a045d36c177676c3af883aed37da3cce61f30b6).